modules/gcve-private-cloud/main.tf

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  region = join("-", slice(split("-", "${var.zone}"), 0, 2))
+  vmwareengine_network = (
+    var.vmwareengine_network_create
+    ? try(google_vmwareengine_network.private-cloud-network.0, null)
+    : try(data.google_vmwareengine_network.private-cloud-network.0, null)
+  )
+  psa_peering = {
+    for k, v in data.google_compute_network_peering.psa_peering : k => slice(split("/", "${v.peer_network}"), 6, 7)[0]
```

Comment:
nits:

- I would break up the line after `:`
- using a regexp might be clearer, or alternatively put a sample value in a comment above the line